ADAL/src/request/ADAuthenticationRequest+AcquireToken.m

```diff
@@ -500,4 +506,19 @@ - (void)requestTokenByCode:(NSString *)code
               completion:completionBlock];
 }
 
+- (void)tryRefreshToken:(ADAuthenticationCallback)completionBlock
+{
+    [[ADTelemetry sharedInstance] startEvent:[self telemetryRequestId] eventName:AD_TELEMETRY_EVENT_ACQUIRE_TOKEN_SILENT];
```

Comment:
Not sure if we want another event name here...

Reply:
Should be fine :)